api/src/pcapi/core/offerers/repository.py

```diff
@@ -211,6 +211,10 @@ def find_offerer_by_validation_token(token: str) -> models.UserOfferer | None:
     return models.Offerer.query.filter_by(validationToken=token).one_or_none()
 
 
+def find_user_offerer_by_id(userOffererId: int) -> models.UserOfferer | None:
+    return models.UserOfferer.query.filter_by(id=userOffererId).one_or_none()
```

Comment:
(détail) Tu peux utiliser `.get()` quand ton filtre ne porte que sur la clé primaire : 

```suggestion
    return models.UserOfferer.query.get(user_offerer_id)
```

... ce qui me fait douter de l'intérêt de la fonction, mais c'est peut-être une question de goût. :)

Reply:
Yes pour le changement !
J'ai gardé la fonction pour rester dans le ton avec les offerers

Reply:
Effectivement. Je me demande pourquoi d'ailleurs. J'ai l'impression que `UserOfferer.id` est la bonne et seule clé primaire nécessaire. Avec la contrainte d'unicité (qui existe déjà) sur le couple `(userId, offererId)`, ce serait suffisant. Je me note de tenter de simplifier ce modèle.